ethclient/ethclient.go

```diff
@@ -101,6 +101,23 @@ func (ec *Client) BlockNumber(ctx context.Context) (uint64, error) {
 	return uint64(result), err
 }
 
+// PeerCount returns the number of p2p peers as reported by the net_peerCount method.
+func (ec *Client) PeerCount(ctx context.Context) (uint64, error) {
+	var result hexutil.Uint64
+	err := ec.c.CallContext(ctx, &result, "net_peerCount")
+	return uint64(result), err
+}
```

Comment:
is this related?

Reply:
No. I tried and `git cherry-pick` picks up these lines for some reason. We might as well keep them.